mock/py/mockbuild/package_manager.py

```diff
@@ -245,16 +246,26 @@ def execute(self, *args, **kwargs):
                 time.sleep(sleep_seconds)
 
             try:
+                str_invocation = " ".join(shlex.quote(arg) for arg in invocation)
+
                 # either it does not support --installroot (microdnf) or
                 # it is bootstrap image made by container with incomaptible dnf/rpm
+
+
                 if not self.support_installroot or self.is_bootstrap_image:
+                    getLog().info("Executing %s in chroot: %s", self.name,
```

Comment:
try calling `self.buildroot.root_log.info()`

Reply:
Good idea!  That may lead to some success ... even though, in case of bootstrap this goes to `<chroot>-bootstrap/results/root.log` file, not `<chroot>/results/root.log`.